cluster/manifests/skipper/deployment.yaml

```diff
@@ -227,12 +227,22 @@ spec:
           {{ end }}
 {{ end }}
         resources:
+{{ if and (eq .Cluster.ConfigItems.enable_dedicate_nodepool_skipper "true") (eq .Cluster.ConfigItems.skipper_ingress_binpack "true") }}
+{{ $cpu_requests := split (printf "%s -%s -%s" .Cluster.ConfigItems.skipper_ingress_cpu .Cluster.ConfigItems.teapot_admission_controller_daemonset_reserved_cpu .Cluster.ConfigItems.kubelet_system_reserved_cpu) " " | sumQuantities }}
+          limits:
+            cpu: "{{ $cpu_requests }}"
+            memory: "{{ .ConfigItems.skipper_ingress_memory }}"
```

Comment:
Should memory be also adjusted by the system `reserved_memory`?

Reply:
This was my argument for not doing it:

> We only substract CPU and not Memory because we run skipper ingress with 1:1 vCPU/GiB ratio and thus there are plenty of space for memory as the c instances has a ratio of 1:2.